CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ way to update this template, but currently, we follow a pattern:
 ---
 ## Upcoming version 2018-08-XX
 
+* [change] Removed the `country` parameter from the search page as it
+  was not used anywhere.
+  [#893](https://github.com/sharetribe/flex-template-web/pull/893)
+
```

Comment:
The changelog could contain even multiple entries for this PR. At least for adding the Mapbox geocoder and maybe also a note about the generalization of the geocoder.

Reply:
Is it ok not to advertise it yet? It's not usable without the script imports, and it would be a violation of the service terms to use the Mapbox geocoder without also using the Mapbox map (which isn't finished). We'll have a big major release when all the parts are ready.

Reply:
Valid point. Let's add an entry when Mapbox can properly be taken into use.